src/containerapp/azext_containerapp/_utils.py

```diff
@@ -839,3 +841,120 @@ def create_acrpull_role_assignment_if_needed(cmd, registry_server, registry_iden
                         raise UnauthorizedError(message) from e
                 else:
                     time.sleep(5)
+
+
+def create_kube_client(kube_config, kube_context, skip_ssl_verification=False):
+    kube_config = normalize_kube_config(kube_config)
```

Comment:
The `kube_config` is not required base on the _params.py setting.
The `normalize_kube_config` function return `Non` if the `kube_config` is None.
We should have validation logic for it, right?